### PR TITLE
Ports/Adds new outfits for command and centcom staff!

### PR DESCRIPTION
**Pull request by Imaginos16**
## About The Pull Request
This PR is the spiritual successor to my second PR ever, #57023, of which it follows a similar concept!

The aim of this PR is to add several new outfits and a new mechanic to go along with them:
-The CentCom Commander Open Coat
-The Head of Security Parade Jacket (Both open and closed variants)
-The Captain's Parade Jacket (open variant)
image

It also recolors the captain's parade jacket to a lighter blue, more suitable with the rest of their aesthetic!
To use the open variants, simply alt-click on the parade jackets to change the style easily on the fly!
The Head of Security Parade Jacket will be easily obtained by accessing the Head of Security closet!

UPDATE:

This PR now resprites the classic captain's hat, the captain's gloves, and the centcom hat, to now be more in line with modern /tg/ shading!

image

It now also resprites the carapace and captain's jumpsuit to be more in line with the captain's hat!

image

Special Thanks to:

@tf-4, for making the original sprites on PR: [#3363](https://github.com/Skyrat-SS13/Skyrat-tg/pull/3363) Of Skyrat
@MrMelbert, for assisting me on ironing out the kinks regarding the code of this PR.
@Twaticus, for giving me the go-ahead to PR this!

And for every lovely person that helped me get this PR off the ground, and gave me the necessary encouragement to improve on the designs!

## Why It's Good For The Game
The aim of this PR is to give even more customization options for any head of staff or centcom staff member worth their salt to enjoy a fancier aesthetic when the moment calls for it! After all, SS13 is not only a game about roleplaying, but also one of cosmetics and fashion!

## Changelog
:cl: PositiveEntropy, tf-4
imageadd: Thanks to a collaboration between the frontier sector and the core sector, the parade jackets now boast new and varied apperances for all to enjoy, with a new parade jacket releasing for the Head of Security!
imageadd: In no short effort than the finest of tailors, the captain's hat, the centcom hat, the captain's jumpsuit and the captain's carapace now have finer gold trims and the finest quality leather available, making them more vibrant for all!
expansion: To utilize the new variants, simply alt-click a parade jacket, or a centcom commander coat. You can also obtain a Head of Security parade jacket by simply accessing the Head of Security's closet!
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
Very sexy, I like it.

**Comment by DAKKA-WAAAGH** [comment]
Very tasteful and stylish! Great job.

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -138,11 +138,18 @@
 	desc = "A sinister looking vest of advanced armor worn over a black and red fireproof jacket. The gold collar and shoulders denote that this belongs to a high ranking syndicate officer."
 	icon_state = "syndievest"
 
-/obj/item/clothing/suit/armor/vest/capcarapace/alt
+/obj/item/clothing/suit/toggle/armor/vest/capcarapace/alt
```
Comment (Fikou) [inline code/modules/clothing/suits/armor.dm:141]: no

Comment (Imaginos16) [reply code/modules/clothing/suits/armor.dm]: fixed

**Review by Krysonism — APPROVED** [review]

**Comment by SweptWasTaken** [comment]
These look good but the captain's hat kinda stands out compared to the uniform.

**Comment by Imaginos16** [comment]
> These look good but the captain's hat kinda stands out compared to the uniform.

Mostly due to how the captain's hat in 4407 was based on the old, bulky armor they used to wear. The centcom hat suffers from a similar issue. Perhaps in a future PR, I'll re-shade them to be in line with more recent aesthetics.

**Comment by Imaginos16** [comment]
> These look good but the captain's hat kinda stands out compared to the uniform.

After debating with myself, I decided to go full steam ahead, and resprite it as you pointed out. Hopefully, its better now!

**Comment by CRITAWAKETS** [comment]
Now that the hat is resprited, how does the carapace look in comparison?

**Comment by EricZilla** [comment]
I am also concerned that now that the hat is toned to match the parade jacket, that it wont look right with the rest of the uniform

**Comment by Imaginos16** [comment]
Hopefully the concerns regarding the carapace and jumpsuit have been cleared. I've also taken the opportunity to improve the gloves, since they had... Problems.

**Review by Twaticus — APPROVED** [review]

**Comment by bluetshirtguy** [comment]
It looks like it's straight up from an anime with how cartoony the tail of the coat is.

**Pull request merged**